storage/migrations/02_runtimes.up.sql

```diff
@@ -3,6 +3,7 @@
 BEGIN;
 
 CREATE TYPE runtime AS ENUM ('emerald', 'sapphire', 'cipher');
+CREATE TYPE call_format AS ENUM ('encrypted/x25519-deoxysii');
```

Comment:
evm_call_format?

Reply:
call formats are actually defined outside of the evm, and they're also used in the whole-transaction "cipher" style encryption. could go with runtime_call_format, if we want to be clear it's not used in consensus. shall we?

Reply:
I'm gonna merge this with the current name